gym/lib/gym/generators/build_command_generator.rb

```diff
@@ -41,6 +41,7 @@ def options
         options << "-derivedDataPath '#{config[:derived_data_path]}'" if config[:derived_data_path]
         options << "-resultBundlePath '#{result_bundle_path}'" if config[:result_bundle]
         options << config[:xcargs] if config[:xcargs]
+        options << "OTHER_SWIFT_FLAGS=\"\$(inherited) -Xfrontend -debug-time-function-bodies\" | grep .[0-9]ms | grep -v ^0.[0-9]ms | sort -nr > culprits.txt" if config[:analyze_build_time]
```

Comment:
@mfurtak `$(inherited)` allows targets to inherit settings from the project level. This is often used for flags and other build settings that are set at the project level and can be fine tuned on the target level.